code-challenges.js

```diff
@@ -49,3 +120,16 @@ var testString2 = "jabberwocky"
 var arr1 = [3, 7, 10, 5, 4, 3, 3]
 var arr2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+
+//The ... syntax allows the function to take in multiple arguments and combine them into a single array called arrays
+const mergeArrays = (...arrays) => {
+    let jointArray = []
+//The forEach loop takes the nested arrays in arrays and combines them into one array called jointArray
+    arrays.forEach(array => {
+        jointArray = [...jointArray, ...array]
+    })
+//uniqueArray uses filter to check if index of the current value matches the index of the current iteration
+    let uniqueArray = jointArray.filter((value,index) => jointArray.indexOf(value) === index)
+    return uniqueArray
```

Comment:
Nice! Consider this- instead of creating a brand new variable and returning it, what if we just returned jointArray.filter(...)?
Example: 

`return jointArray.filter((value,index) => jointArray.indexOf(value) === index)`